### ISSUE TITLE
Remove deprecated twisted.protocols.telnet

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8925]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8925|
|Type|enhancement|
|Created|2016-11-26 22:50:10Z|
|Branch|https://github.com/twisted/twisted/tree/8925-tp-telnet|

<details><summary>Searchable metadata</summary>

```
trac-id__8925 8925
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__8925_tp_telnet 8925-tp-telnet
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1480200610907697 1480200610907697
changetime__1480715699060229 1480715699060229
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

PR available at https://github.com/twisted/twisted/pull/609

**Comment by twisted-trac** [comment]
|[<img alt="derwolfe's avatar" src="https://avatars.githubusercontent.com/u/641357?s=50" width="50" height="50">](https://github.com/derwolfe)<a name="note_3"></a>|@derwolfe set owner to @hawkowl|
|-|-|

Please see my review on github (https://github.com/twisted/twisted/pull/609#pullrequestreview-10555978)

Thanks for working on this!

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set status to `closed`|
|-|-|

In changeset 83d2790429d2fc4b9e8c684ae4ec48b6a22a3add
```
#!CommitTicketReference repository="" revision="83d2790429d2fc4b9e8c684ae4ec48b6a22a3add"
Merge 8925-tp-telnet: remove deprecated twisted.protocols.telnet

Author: hawkowl

Reviewer: derwolfe

Fixes: #8925
```
